### PR TITLE
Update machine-learning.md

### PR DESCRIPTION
**Pull request by Pratikdate**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [# ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Pratikdate** [comment]
Please add hactoberfest-accepted lable

File: content/ai/concepts/machine-learning/machine-learning.md [hunk 1]
```diff
@@ -62,6 +62,14 @@ Deep Learning is the subset of machine learning that focuses on artificial neura
 
 The relatively high performance of neural networks in human-like tasks such as speech and object recognition distinguish deep learning from other machine learning algorithms.
 
+
```
Comment (SSwiniarski) [inline content/ai/concepts/machine-learning/machine-learning.md:65]:
```suggestion
```
Remove blank line

File: content/ai/concepts/machine-learning/machine-learning.md [hunk 2]
```diff
@@ -62,6 +62,14 @@ Deep Learning is the subset of machine learning that focuses on artificial neura
...
+## Fine-Tuning
+
+Based on the evaluation results, you might fine-tune the model's hyperparameters or adjust the model architecture to improve its performance.There are several methods to fine-tune models.
```
Comment (SSwiniarski) [inline content/ai/concepts/machine-learning/machine-learning.md:68]:
```suggestion
Based on the evaluation results, the model's user might fine-tune the model's hyperparameters or adjust the model architecture to improve its performance. There are several methods to fine-tune models.
```
We avoid first- and second-person pronouns in Docs.

File: content/ai/concepts/machine-learning/machine-learning.md [hunk 3]
```diff
@@ -62,6 +62,14 @@ Deep Learning is the subset of machine learning that focuses on artificial neura
...
+  - Hyperparameter Tuning:Hyperparameters are configuration settings for a machine learning model that are not learned during training but are set before training begins. These settings can significantly impact a model's performance.
```
Comment (SSwiniarski) [inline content/ai/concepts/machine-learning/machine-learning.md:69]:
```suggestion
  - Hyperparameter Tuning: Hyperparameters are configuration settings for a machine learning model that are not learned during training but are set before training begins. These settings can significantly impact a model's performance.
```

File: content/ai/concepts/machine-learning/machine-learning.md [hunk 4]
```diff
@@ -62,6 +62,14 @@ Deep Learning is the subset of machine learning that focuses on artificial neura
...
+  - Model Architecture Adjustments:There are several methods to accurately adjust the Architecture of ML algorithms.There are some technique such as Feature Engineering,Model Selection,Ensemble Methods .
```
Comment (SSwiniarski) [inline content/ai/concepts/machine-learning/machine-learning.md:70]:
```suggestion
  - Model Architecture Adjustments: There are several methods to accurately adjust the Architecture of ML algorithms. There are techniques such as Feature Engineering, Model Selection, and Ensemble Methods .
```

File: content/ai/concepts/machine-learning/machine-learning.md [hunk 5]
```diff
@@ -62,6 +62,14 @@ Deep Learning is the subset of machine learning that focuses on artificial neura
...
+
```
Comment (SSwiniarski) [inline content/ai/concepts/machine-learning/machine-learning.md:71]:
```suggestion
```
Remove blank line.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Pratikdate I did a first review and left a few comments.

**Review by SSwiniarski — APPROVED** [review]
@Pratikdate Looks good for a second review.

**Review by KTom101 — APPROVED** [review]
@Pratikdate I added some minor changes directly, there were a couple of things in the original content I cleaned up. I also ran `npm run format:all` to fix the failed [checks](https://www.codecademy.com/resources/docs/contribution-guide/getting-set-up-on-your-computer#:~:text=There%20are%20continuous%20integration%20(CI)%20workflows).

**Pull request merged**